docs/spec/v1alpha1/receivers.md

```diff
@@ -0,0 +1,52 @@
+# Receivers
+
+The `Receiver` API defines a webhook receiver that triggers
+a synchronization for a group of sources.
+
+## Specification
+
+```go
+type ReceiverSpec struct {
+	// Type of webhook sender, used to determine
+	// the validation procedure and payload deserialization.
+	// +kubebuilder:validation:Enum=github;gitlab
+	// +required
+	Type string `json:"type"`
```

Comment:
Do we also want to declare the enums as `const`s?

Reply:
Done